### PR TITLE
vim: add :wq, in case of emergency for beginners.

### PR DESCRIPTION
**Pull request by juniorRubyist**
- [ ] The page has 8 or fewer examples.
> But the one added seems necessary.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/vim.md
Comment (navarroaxel) [inline pages/common/vim.md:23]: We prefer the long format to be easier for beginners.

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages/common/vim.md
Comment (juniorRubyist) [reply pages/common/vim.md:23]: That's a fair point.

File: pages/common/vim.md
Comment (sbrl) [inline pages/common/vim.md:7]: Normally, I would suggest that this is not part of the standard format for tldr pages, and should be removed. However, this is a very unusual page, and it adds value. HTML support though isn't in the [client spec](https://github.com/tldr-pages/tldr/blob/master/CLIENT-SPECIFICATION.md), so it'll have to be removed. (CommonMark support is in the spec though, hence why I support leaving this in).
```suggestion
**Important:** If you accidentally open Vim and have never used it before, _do not_ press anything. Type `:q!` and press enter to quit.
```

Hopefully we can prevent situations such as [this one](https://www.pixelstech.net/images/fun/normal/IMG-20170805-WA0000.jpg) :P ([source](https://www.pixelstech.net/fun/412-Exit-Vim))

**Review by sbrl — CHANGES_REQUESTED** [review]
Thanks for the edit, @juniorRubyist ! I've left some comments below for you to review.

Also, we should wait for feedback from others before merging this due to the unusual situation here.

**Comment by juniorRubyist** [comment]
@navarroaxel @sbrl I have implemented your suggestions. Let me know if any other changes may be needed.

**Review by sbrl — COMMENTED** [review]
Looks ok to me! We should get some additional opinions before merging though, so let's wait a bit to give the others time to review.

File: pages/common/vim.md
Comment (sbrl) [inline pages/common/vim.md:7]: Oops! I forgot to return to this PR. Sorry about that! Upon further investigation, it looks like this line is causing the build to fail, so we probably need to remove it.

Comment (navarroaxel) [inline pages/common/vim.md:7]:
```suggestion
> **Important:** If you accidentally open Vim and have never used it before, _do not_ press anything. Type `:q!` and press enter to quit.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Comment by bl-ue** [comment]
@juniorRubyist thank you very much for this PR, however, I don't think we need another example as the 3rd and 4th examples already show how to save and quit, respectively.  We try to keep a maximum of 8 examples.

**Comment by bl-ue** [comment]
@juniorRubyist Just a note: the title of this PR reads "add :wq, in case of emergency for beginners", but in the event that a user is accidentally stuck in vim and wants out, it's very unlikely they want to write the files (even if they didn't change them it will still update the date on the file) 😉

**Pull request closed without merging**
